compiler/rustc_lint/src/non_ascii_idents.rs

```diff
@@ -314,8 +309,6 @@ impl EarlyLintPass for NonAsciiIdents {
                 let mut lint_reports: BTreeMap<(Span, Vec<char>), AugmentedScriptSet> =
                     BTreeMap::new();
 
-                // The end result is put in `lint_reports` which is sorted.
-                #[allow(rustc::potential_query_instability)]
```

Comment:
This allow is ok, as there is an explaining comment with it. Or remove the sort?